services/app-web/src/pages/LinkYourRates/LinkRateSelect.tsx

```diff
@@ -135,13 +140,22 @@ export const LinkRateSelect = ({
             ?.toLowerCase()
             .includes(input.toLowerCase())
 
+    //We track rates that have already been selected to remove them from the dropdown
+    const selectedRates = values.rateForms.map((rate) => rate.id && rate.id)
```

Comment:
@ruizajtruss I'm not seeing this code working in the review app. Down to  pair together - I could be missing something.

I started to wonder if it is just not r-rendering the options list when the values change